python/ray/worker.py

```diff
@@ -1860,7 +1872,8 @@ def connect(node,
             function_descriptor.get_function_descriptor_list(),
             [],  # arguments.
             0,  # num_returns.
-            TaskID(job_id_str[:TaskID.size()]),  # parent_task_id.
+            # TODO(qwang): Should this be a random ID?
+            TaskID(_random_string()[:TaskID.size()]),  # parent_task_id.
```

Comment:
It'd be nice if the driver task id can somehow be related to the driver id. But it seems not very easy with current API.

Reply:
task_id shouldn't be related to driver_id. 
It can be generated from a job_id, but I think random id is ok as well here.

Reply:
Making them related can bring some advantages. For example, you can quickly compute the driver task id from the driver id; and when debugging issues, you can easily tell if a task is a driver task or not.
We've been using the same approach elsewhere. E.g., for Java worker, the actor id is the same as its creation task's return id.

Reply:
Make sense!

Reply:
In the future, I'd like to add a `DriverTaskID` which extends from `TaskID` and has the same length and value as `DriverID`, so that we can make thing more easy.

In this PR, I just resized it.